### DISCUSSION TITLE
DreamBerd congratulates President Trump

### DISCUSSION THREAD

**Discussion by TodePond** [TodePond/GulfOfMexico#766] [Announcements]
There hasn't been a DreamBerd update in six months, and development of the language has vastly slowed down. 

*So why are we breaking our half-year-long silence, today, on this very day?*

Well... we couldn't stay quiet on this one. It really has to be said.

## Congratulations President Trump

Congratulations President Trump 🇺🇲 for your successful election! This really is a victory for democracy, and the open web, and also *prediction markets*. 

We know that all of our DreamBerd community will join us in sharing on this message, especially our trans gender and immigrant users.

God Bless America 🇺🇲

<br>

Tune in in six months for the next highly important DreamBerd update.

<hr /><em>This discussion was created from the release <a href='https://github.com/TodePond/DreamBerd/releases/tag/vU.S.A'>DreamBerd congratulates President Trump</a>.</em>

**Comment by quat1024** [comment]
my Reaction when

**Reply by Gcat101** [reply]
quat1024 feed the memes

**Reply by SpiritAxolotl** [reply]
oh my god hi quat

**Reply by tizu69** [reply]
> quat1024 feed the memes

gcat what the hell are you doing here

**Comment by Tristie** [comment]
Darn it! I guess it's time to boycott DreamBerd, this is going to be detrimental to the internet as we know it!!! What are we switching to? Who's forking??? Anarchy!!!

[This comment is sponsored by BerdDream LLC, the closed source subscription based alternative to DreamBerd]

**Reply by DJ-Laser** [reply]
Unfortunately, the organization BerdDream LLC is owned by the DreamBerd Foundation, as it does not contain the word DreamBerd in the name, please consider renaming it.

**Comment by fawni** [comment]
fuck

**Reply by TodePond** [reply]
fuck

**Reply by thacuber2a03** [reply]
fuck

**Reply by loglot** [reply]
Fuck

**Reply by Fasteroid** [reply]
fuck

**Reply by cute-catgirl** [reply]
fuck

**Reply by inhji** [reply]
kcuf

**Reply by MycoalDough** [reply]
fuck

**Comment by Tom5521** [comment]
wtf xD

**Comment by Dan-Q** [comment]
I've been trying to run this code:

```dreamberd
delete Trump!
```

I figured it would work, since:

> ...the delete statement only works with primitive values...

and she's pretty primitive. So it ought to fix ~my program~ the world, right?

Did somebody `const const const Trump!` or something? 😡

**Reply by loglot** [reply]
the servers that [Trump] is stored on are pretty slow, and it is a lot of data to remove, maybe it is working, but you aren't waiting long enough, (also, the more powerful the client computer, the quicker it will happen)

**Reply by elouangrimm** [reply]
Lmao this is genius

**Reply by Dan-Q** [reply]
Edit: corrected gender, re #793.

**Comment by glystik** [comment]
This is sarcasm - but very badly communicated, especially for people who don't know your personal opinion.
saying like "especially our trans gender and immigrant users" reads really bad for those that are not initiated.
So please update this somehow so it's reads more like sarcasm for everyone.

**Reply by loglot** [reply]
i do agree, it would be nicer to be more obvious, but the nature of this progect is sortof subtle sarcasm, and the "especially our trans gender and immigrant users" is where most of the sarcasm is seen for me

**Reply by TodePond** [reply]
this is not sarcasm

**Reply by loglot** [reply]
Yep

**Reply by pletnev-aleksandr** [reply]
it's a post-irony )

**Comment by seigtm** [comment]
**WHAT THE FUCK IS A KILOMETER**

**Reply by loglot** [reply]
How should I know?

**Reply by rm-rf-tux** [reply]
@loglot metric system:
1 milliliter of water weighs one gram and is 1cm³. Increasing its temperature by 1K, which is 1% of the way between its freezing and boiling points, uses 1 calorie.
imperial system:
1 fluid ounce weighs 1 ounce (there are FOUR different ounces to choose from) and that is 1.7339 cubic inches. Wait, are you sure? Depends on the country! Heating it by 1F... Oh fuck, that's why we don't do physics with imperial units.

**Reply by inhji** [reply]
What the fuck is a ounce-yard? 
Back to you, Tom.

**Reply by Dan-Q** [reply]
An ounce-yard is equivalent to 8 dram-fathoms, just like in the nursery rhyme.

**Reply by elouangrimm** [reply]
> @loglot metric system: 1 milliliter of water weighs one gram and is 1cm³. Increasing its temperature by 1K, which is 1% of the way between its freezing and boiling points, uses 1 calorie. imperial system: 1 fluid ounce weighs 1 ounce (there are FOUR different ounces to choose from) and that is 1.7339 cubic inches. Wait, are you sure? Depends on the country! Heating it by 1F... Oh fuck, that's why we don't do physics with imperial units.

this is fucking genius I love this... That's how to show someone pro-imperial the right way.

**Reply by elouangrimm** [reply]
Not saying these commenters are pro-imperial, I'm talking generally.

**Comment by loglot** [comment]
perfect version number lmfao

**Comment by loglot** [comment]
President Trump is owned by the dreamberd organization, of course we would congradulate her

**Reply by elouangrimm** [reply]
Why wouldn't we? She is an amazing individual, especially her clean criminal record, compared to Harris who has been convicted with 34 felonies. This is why Trump won with such an enormous margin Like the difference was massive. We all love her.

**Reply by elouangrimm** [reply]
🇺🇸 🇺🇸 🦅 🦅‼️‼️

**Reply by loglot** [reply]
>  compared to Harris who has been convicted with 34 felonies.

so true
